compose/foundation/foundation/src/skikoTest/kotlin/androidx/compose/foundation/gestures/DragGestureTest.kt

```diff
@@ -313,6 +313,11 @@ class DragGestureTest {
             assertFalse(dragCanceled)
             assertFalse(dragEnded)
 
+            scene.sendPointerEvent(
```

Comment:
This is sent because we test a drag to `Offset(5f + viewConfiguration.touchSlop, 15f),` position. But because it is different from the previous Move position and we don't send a synthetic now, we send a real Move now